src/pages/Map.tsx

```diff
@@ -128,6 +128,7 @@ export const MapPage = (): JSX.Element => {
           attributionControl={false}
           renderWorldCopies={false}
           maxPitch={0}
+          style = {{filter: darkMode ? 'invert(100%)' : ''}}
```

Comment:
This method of flipping all colors is less preferable (imo) than changing the style json on line 115 which supports changing the background RGB.

I found two full style sets online (sourced from https://github.com/MacroGrove/maplibre-gl-darkmode-plugin/blob/ef4277e6581c03e36a647bd66d46a03055de0634/static/scripts/maplibre-gl-darkmode-plugin.js#L12) that can be toggled between instead:
- Lightmode https://tiles.stadiamaps.com/styles/alidade_smooth.json
- Darkmode https://tiles.stadiamaps.com/styles/alidade_smooth_dark.json

Reply:
This would require us switching off of using OSM for our maps, the source for those style files use https://stadiamaps.com as the source for the tiles. This allows for the styling change using background RGB and such.

See: https://github.com/meshtastic/web/issues/199#issuecomment-2154709194